### PR TITLE
Add ESM+CJS builds and unit tests

### PR DESCRIPTION
**Pull request by donmccurdy**
Trying to keep this fairly simple since there isn't much code here, but if we have other preferences on the tooling (currently ESBuild, and node.js assertions for tests) I'm happy to change it! 😄

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,34 +1,45 @@
 {
   "name": "cartocolor",
   "version": "4.0.1",
-  "description": "A module of CartoColors for browserify",
+  "description": "CARTO custom color palettes for use with maps",
```
Comment (VictorVelarde) [inline package.json]: @donmccurdy The truth is that colors are indeed used now for some other things (eg. bold palette in design system for certain elements such as charts or even general UI things, like labels). I would probably suggest to keep in naming original "CartoColors", but dropping the browserify part... or a mix between both.... eg: "CartoColors: custom color palettes from CARTO"

**Review by VictorVelarde — APPROVED** [review]
👍🏻

File: src/index.js
Comment (donmccurdy) [inline src/index.js:1]: TODO: Test the new ESM exports in `@deck.gl/carto`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Version 5.0.0
+Released 2024-05-13
+
+- Update license to [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/)
+- Add ESM and CJS builds
```
Comment (donmccurdy) [inline CHANGELOG.md:7]: Another change here is that we don't export unknown colorbrewer palettes. We should do one of the following:

- (A) Document that only known/tagged colorbrewer palettes are exported
- (B) Check which palettes are missing, tag and add them

Comment (donmccurdy) [reply CHANGELOG.md:7]: Chose (B), done.

File: package.json (hunk 1 above)
Comment (donmccurdy) [reply package.json]: Sounds good, done!

File: src/index.js
Comment (donmccurdy) [reply src/index.js:1]: Done.

**Pull request merged**